vertx-web/src/main/java/io/vertx/ext/web/handler/impl/AuthenticationHandlerImpl.java

```diff
@@ -111,6 +112,10 @@ public void handle(RoutingContext ctx) {
     });
   }
 
+  protected void processSuccessAuth(RoutingContext ctx) {
```

Comment:
Perhaps we could move this method to the interface `AuthenticationHandler` and call it:

`postAuthentication(RoutingContext)`

The docs should state that this method is called to perform any `post` authentication tasks, such as redirects. By default it calls: `context.next()` and it should be called in the line you changed above + 
https://github.com/vert-x3/vertx-web/pull/1719/files#diff-25e566419b93c44fc9ae137620b4007fR81

Reply:
Done